protected/humhub/modules/marketplace/Events.php

```diff
@@ -8,10 +8,19 @@
 
 namespace humhub\modules\marketplace;
 
+use humhub\components\Module as CoreModule;
+use humhub\modules\admin\events\ModulesEvent;
+use humhub\modules\admin\libs\HumHubAPI;
+use humhub\modules\admin\widgets\ModuleControls;
+use humhub\modules\admin\widgets\ModuleFilters;
+use humhub\modules\admin\widgets\Modules;
+use humhub\modules\marketplace\models\Module as ModelModule;
+use humhub\modules\ui\icon\widgets\Icon;
+use humhub\modules\ui\menu\MenuLink;
+use humhub\widgets\Button;
 use Yii;
 use yii\base\BaseObject;
 use yii\base\Event;
-use yii\helpers\Url;
 
 class Events extends BaseObject
```

Comment:
In the whoile file we should check the `Module::$enabled` is `true` before handling the Events.

Reply:
https://github.com/humhub/humhub/pull/5476/commits/910bb0c77beed80b6af022e185b1cc42feca7a9f